src/orquestra/quantum/operators/_pauli_operators.py

```diff
@@ -549,6 +549,12 @@ def __hash__(self):
     def is_constant(self) -> bool:
         return len(self.terms) == 0 or all([term.is_constant for term in self.terms])
 
+    @property
+    def constant_value(self) -> Union[int, float]:
+        return sum(
+            [term.coefficient for term in self.terms if term.is_constant]  # type:ignore
```

Comment:
I don't like this `type:ignore` 🙃

Reply:
I spent little over an hour dealing with this MyPy issue. @dexter2206?